ompi/mpi/fortran/mpif-h/Makefile.am

```diff
@@ -111,7 +111,7 @@ CLEANFILES += sizeof_f.f90
 # out this workaround indefinitely.  So Jeff advised Absoft to fix
 # this bug in Sep 2014.
 noinst_LTLIBRARIES=
-if BUILD_FORTRAN_SIZEOF
+if BUILD_PMPI_FORTRAN_MPIFH_BINDINGS_LAYER
```

Comment:
1. yes
3. looks better than 2.
I will revamp the fix, but unlikely before Friday